.pylint-spelling-private-dict

```diff
@@ -0,0 +1,119 @@
+abspath
```

Comment:
Can we move this file to somewhere else - config/ or data/?
This filename doesn't seem to be required by pylint, I imagine we'll need to edit it fairly regularly, and I don't see the need for it to be in the project root.

Reply:
This will be project dependent I can move it to `config/pylint/` if you think that is a more appropriate location.